qa-pipelines/tasks/cf-deploy.sh

```diff
@@ -30,31 +30,31 @@ fi
 echo UAA customization ...
 echo "${HELM_PARAMS[@]}" | sed 's/kube\.registry\.password=[^[:space:]]*/kube.registry.password=<REDACTED>/g'
 
-# Deploy UAA
-kubectl create namespace "${UAA_NAMESPACE}"
-if [[ "${PROVISIONER}" == "kubernetes.io/rbd" ]]; then
-    kubectl get secret -o yaml ceph-secret-admin | sed "s/namespace: default/namespace: ${UAA_NAMESPACE}/g" | kubectl create -f -
-fi
+if [ "${EMBEDDED_UAA:-false}" != "true" ]; then
```

Comment:
[[ ]] for consistency

Reply:
Fixed in 86a2d8d13026096aad7fecfcb1d52bb81fe4f16e.